sntp: add page (#11877)
tldr-pages/tldr: pages/osx/sntp.md

# sntp

> > A very Simple Network Time Protocol client program.

Reviewer: ```suggestion
> A very Simple Network Time Protocol client program.
> More information: <https://keith.github.io/xcode-man-pages/sntp.1>.
```

> - Query SNTP server {{pool.ntp.org}} and display the time:

Reviewer 2: ```suggestion
- Query the SNTP server `pool.ntp.org` and display the time:
```

`sntp {{pool.ntp.org}}`

> - Synchrize the system clock with {{pool.ntp.org}} SNTP server:

Reviewer 2: ```suggestion
- Synchronize the system clock with the SNTP server at `pool.ntp.org`:
```

`sudo sntp -S {{pool.ntp.org}}`

> - Enabe debug logging:

Reviewer 2: ```suggestion
- Enable debug logging:
```
